package.yaml

```
@@ -49,8 +49,8 @@ library:
     - -W
 
 tests:
-  parser:
-    main: Parser.hs
+  spec:
```

Minor: if we have one test suite we can follow the typical(?) naming convention for test suites:
```suggestion
  wai-saml2-test:
```